src/components/badge/Badge.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import classNames from 'classnames';
+
+import './Badge.scss';
+
+type Props = {
+    /** Child components */
+    children: React.ReactNode;
+    /** A predefined badge type */
+    className?: string;
+    /** Adds a class to the component */
+    type?: 'default' | 'info' | 'warning' | 'highlight' | 'error' | 'alert' | 'success';
```

Comment:
Could we convert this to an enum, similar to `ButtonType` in Button.tsx?

Reply:
Migrated to `enum`! https://github.com/box/box-ui-elements/pull/2535/commits/5b8736195ec22cf75f52cf8b411d04695c8f285f

I also exported that enum out via `index.ts` since I think that's necessary iirc.